### PR TITLE
Remove obsolete text and OT information

### PR DESCRIPTION
**Pull request by hoch**
PTAL. This PR removes the obsolete text and the origin trial information for the M66 stable launch.

### CODE REVIEW

File: audio-worklet/index.html [hunk 1]
```diff
@@ -36,33 +34,11 @@ <h1>
...
-        <pre><code>chrome://flags/#enable-experimental-web-platform-features</code></pre>
-        <p>
-          If the status indicator on the top-right corner of this page is not
-          green, it means either your browser does not support AudioWorklet
-          or you need to follow the instruction above to activate the
-          feature.
+          target="_blank">AudioWorklet</a> is available on <strong>Chrome 66 (or
+          later) by default</strong>. If the status indicator on the top-right
+          corner of this page is not green, it means either your browser does
+          not support AudioWorklet or you need to update Chrome browser to
```
Comment (rtoy) [inline audio-worklet/index.html]: I'd rephrase this slightly:

it means your browser does not support currently AudioWorklet.  If you are using Chrome, please update to Chrome 66.

**Comment by hoch** [comment]
Thanks, done and merging.
